addons/web/static/src/js/views/basic/basic_model.js

```diff
@@ -2434,11 +2434,19 @@ var BasicModel = AbstractModel.extend({
                     const modelFieldName = record.fields[modelField].string;
                     const referenceFieldName = record.fields[fieldName].string;
 
-                    this.do_warn(_t(`'${referenceFieldName}' is unsynchronized
-                                     with '${modelFieldName}'.`),
-                                 _t(`If you change ${modelFieldName} or
-                                     ${referenceFieldName}, the synchronization
-                                     will be reapplied and the data will be modified.`), true);
+                    this.do_warn(
+                        _.str.sprintf(
+                            _t(`'%s' is unsynchronized with ''.`),
+                            referenceFieldName,
+                            modelFieldName,
+                        ),
```

Comment:
Hello, I believe the second `%s` is missing from here

Reply:
ah shoot, good catch.

https://github.com/odoo/odoo/pull/68320